Framework/DataObjects/inc/MantidDataObjects/MDHistoWorkspace.h

```diff
@@ -426,7 +426,7 @@ class DLLExport MDHistoWorkspace : public API::IMDHistoWorkspace {
   }
 
   MDHistoWorkspace *doCloneEmpty() const override {
-    return new MDHistoWorkspace(0);
```

Comment:
Looking at the constructor signatures I think this might be clearer if we just take away the null pointer so it's more obvious it's using the default constructor?

Reply:
As discussed offline I misread the constructor as having all arguments defaulted - this change is fine.